src/redux/actions/index.js

```diff
@@ -34,3 +35,10 @@ export function setLanguageName(language) {
     payload: language,
   };
 }
+
+export function setSecondLanguage(language) {
+  return {
+    type: SET_SECOND_LANGUAGE,
+    payload: language,
+  };
+}
```

Comment:
remove this and use existing setLanguageName then read the value of it in any component to decide on what lang version should be displayed

Reply:
Why was this resolved?

Reply:
I have deleted these lines from my code. That's why I said it is resolved.